src/SmsLength.php

```diff
@@ -162,12 +162,17 @@ private function inspect(string $messageContent): void
         // Any character outside the 7-bit alphabet switches the entire encoding to UCS-2
         $this->encoding = '7-bit';
         $this->size = 0;
+
         $mbLength = mb_strlen($messageContent, 'UTF-8');
         for ($i = 0; $i < $mbLength; $i++) {
             $char = mb_substr($messageContent, $i, 1, 'UTF-8');
             if (in_array($char, self::GSM0338_BASIC, true)) {
                 $this->size++;
             } elseif (in_array($char, self::GSM0338_EXTENDED, true)) {
+                // In cases where a double counted char straddles two messages, add padding to push it to the next part
```

Comment:
I think it may be clearer to say *why* we need to count as 3; something like:
> In cases where an extended char would straddle two messages, a padding control character is added first,
> so the extended character is pushed entirely to the next part".